xcb-types.el

```diff
@@ -456,6 +461,66 @@ Consider let-bind it rather than change its global value."))
             (throw 'break 'nil)))))
     result))
 
+
+(defsubst xcb:-f64-to-binary64 (value)
```

Comment:
There's some code duplication here, and lots of constants:

1. Abstracting over f32/f64 made reading the code impossible.
2. The constants might as well be magic numbers. The calculations weren't any easier to understand.

Reply:
Doesn't look that bad. The constants look familiar.